enterprise/payara.eecommon/nbproject/org-netbeans-modules-payara-eecommon.sig

```diff
@@ -1265,8 +1265,8 @@ meth public abstract org.netbeans.modules.j2ee.deployment.common.api.Datasource
 meth public abstract org.netbeans.modules.j2ee.deployment.common.api.MessageDestination createMessageDestination(java.lang.String,org.netbeans.modules.j2ee.deployment.common.api.MessageDestination$Type) throws org.netbeans.modules.j2ee.deployment.common.api.ConfigurationException
 meth public final org.netbeans.modules.j2ee.dd.api.common.RootInterface getStandardRootDD()
 meth public final org.netbeans.modules.j2ee.dd.api.webservices.Webservices getWebServicesRootDD()
-meth public final static org.openide.util.Pair<java.io.File,java.lang.Boolean> getExistingResourceFile(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.tooling.data.PayaraVersion)
-meth public final static org.openide.util.Pair<java.io.File,java.lang.Boolean> getNewResourceFile(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.tooling.data.PayaraVersion)
+meth public final static org.openide.util.Pair<java.io.File,java.lang.Boolean> getExistingResourceFile(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.tooling.data.PayaraPlatformVersionAPI)
```

Comment:
Keep old methods, introduce new ones. As the methods differ in parameter type, their overloaded versions shall easily coexist.